apps/xprof_gui/priv/src/actions/NavigationActions.js

```diff
@@ -22,8 +22,9 @@ const setPosition = position => ({
   position,
 });
 
-const clearFunctionBrowser = () => ({
+export const clearFunctionBrowser = () => ({
   type: types.CLEAR_FUNCTION_BROWSER,
+  position: -1,
```

Comment:
field `position` from this action payload isn't used in reducer (or in any other place).
Anyway don't use 'hardcoded' payload in action creator function, Better place is reducer:
```js
    case types.CLEAR_FUNCTION_BROWSER:
      return {
        ...state,
        query: '',
        functions: [],
        position: -1          <----------
      };
```

Reply:
👍